src/pages/Oh/Login/OhLogin.js

```diff
@@ -0,0 +1,94 @@
+import './OhLogin.scss';
+import { useNavigate } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+
+function Login() {
+  const [inputId, setInputId] = useState('');
+  const [inputPw, setInputPw] = useState('');
+  const navigate = useNavigate();
+
+  const handleIdInput = e => {
+    setInputId(e.target.value);
+  };
+
+  const handlePwInput = e => {
+    setInputPw(e.target.value);
+  };
+
+  const handleActive = inputId.includes('@') !== -1 && inputPw.length >= 5;
```

Comment:
```suggestion
  const handleActive = inputId.includes('@') && inputPw.length >= 5;
```
- 함수가 아닌 변수로 작성해주신 것은 너무 좋았습니다.
- !== -1은 결국 true를 생각하신 것이기 때문에 위 코드로 진행해도 될 것 같습니다.